tests/core/test_source.py

```diff
@@ -105,6 +105,18 @@ def test_import_source():
     bt.Gene.import_source(organism="saccharomyces cerevisiae")
     assert bt.Gene.filter(organism__name="saccharomyces cerevisiae").exists()
 
+    # import organism data without 'parents' column (should not crash with KeyError)
+    bt.Organism.import_source()
+    assert bt.Organism.filter().count() > 0
+
+    # test that string sources raise TypeError
+    with pytest.raises(TypeError):
+        bt.CellLine.import_source(source="depmap")
+
+    # test that ICD Source does not crash
```

Comment:
It's good that we're testing this now, but it'd be unclear to other developers why exactly ICD or this version.

What I had suggested is that we find an efficient way to test ALL `import_source` functions. So all sources but of course only their latest versions. Could you please try to find a way to make that happen? As I wrote in the issue, it might require a set up to parallelize that or a new job or something.